roles/front_desk.py

```
@@ -436,7 +436,11 @@ async def _call_secretary_and_execute(self, event: AstrMessageEvent, chat_id: st
                         f"AngelHeart[{chat_id}]: 上游唤醒聊天事件未获批准，已停止后续主 LLM 处理。"
                     )
                     event.stop_event()
-                # 决策不需要回复，立即释放门锁（设置较短的“不回复”冷却）
+                # 决策不需要回复，记录原因并立即释放门锁（设置较短的“不回复”冷却）
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)